packages/cli/lib/validators/__tests__/DependencyValidator.js

```diff
@@ -8,22 +8,10 @@ const { THEME_PATH } = require('./validatorTestUtils');
 jest.mock('fs-extra');
 jest.mock('@hubspot/cli-lib/api/marketplace');
 
-const getMockValidationResult = customPath => {
+const getMockDependencyResult = (customPaths = []) => {
   const result = {
-    renderingErrors: [
```

Comment:
Does this mean no more rendering errors? Only dep checks?

Reply:
Yep - everything is lumped into the one response field on our end